packages/backend/src/core/SequenceProcessor.ts

```diff
@@ -219,4 +218,10 @@ export class SequenceProcessor extends EventEmitter {
     activityLatest.labels({ project: this.id }).set(state.latest)
     this.state = state
   }
+
+  // WARNING: this method should be used only in tests
+  stopQueue(): void {
```

Comment:
Use `/**` for doc comments. This comment won't show up on hover. Also you could make it obvious from the method name e.g. `_TEST_ONLY_stopQueue()`